src/main/resources/assets/paths/institutionById.yaml

```diff
@@ -0,0 +1,67 @@
+get:
+  summary: Institution GET
+  description: Fetches Institution based on id param
+  tags:
+    - Institutions
+  parameters:
+    - name: id
+      in: path
+      description: Institution ID
+      required: true
+      schema:
+        type: string
+  responses:
+    200:
+      description: Returns target institution and its signing officials.
+        Admin will see all details while non-Admins will only see a subset.
+    404:
+      description: Institution not found
+    500:
+      description: Internal Server Error
+put:
+  summary: Institution UPDATE
+  description: Updates institution based on id and json payload
+  tags:
+    - Institutions
+  parameters:
+    - name: id
+      in: path
+      description: Institution ID
+      required: true
+      schema:
+        type: string
+  requestBody:
+    description: Required JSON payload representing updated Institution
+    required: true
+    content:
+      application/json:
+        schema:
+          $ref: '#/components/schemas/Institution'
```

Comment:
We could add `signingOfficials` as a field in `Institution`. Additionally, since we're moving the path file here (nice work!) we could refer to the yaml directly.
```suggestion
          $ref: '../schemas/Institution.yaml'
```

Reply:
Since `PUT api/institutions/{id}` displays the `Institution` schema for the request body but does not use `signingOfficials` as part of the institution update, is it best to 
1) not add `signingOfficials` as a field to `Institution` schema 
2) include `signingOfficials` as a field anyway
3) have separate `Institution` schemas: one with `signingOfficials`, one without

Reply:
Great observation. I would add it as a field, but would add to the description that it is not an updatable field. We have this problem in some other cases as well.